common-util/Contracts/index.jsx

```diff
@@ -1,239 +1,148 @@
 import Web3 from 'web3';
-import { ethers } from 'ethers';
-import { getChainId } from '@autonolas/frontend-library';
+import { LOCAL_FORK_ID } from '@autonolas/frontend-library';
+
+import { LOCAL_CHAIN_ID } from 'util/constants';
+import { getChainId, getProvider } from 'common-util/functions';
 import {
-  // depository
   DEPOSITORY,
-
-  // dispensers
   DISPENSER,
-
-  // treasury
   TREASURY,
-
-  // tokenomics
   TOKENOMICS,
-
-  // bond calculator
   BOND_CALCULATOR,
-
-  // uniswap
   UNISWAP_V2_PAIR_ABI,
-
-  // registries - agent
   AGENT_REGISTRY,
-
-  // registries - component
   COMPONENT_REGISTRY,
-
-  // registries - service
   SERVICE_REGISTRY,
-
-  // erc20
   ERC20_ABI,
 } from 'common-util/AbiAndAddresses';
-import { LOCAL_CHAIN_ID, LOCAL_FORK_ID } from 'util/constants';
-/**
- * Addresses fetched when backend connected locally
- * (initDeploy.json in backend repository)
- */
-export const LOCAL_ADDRESSES = {
-  DEPOSITORY_ADDRESS_LOCAL: '0x4c5859f0F772848b2D91F1D83E2Fe57935348029',
-  DISPENSER_ADDRESS_LOCAL: '0x1291Be112d480055DaFd8a610b7d1e203891C274',
-  TREASURY_ADDRESS_LOCAL: '0x36C02dA8a0983159322a80FFE9F24b1acfF8B570',
-  TOKENOMICS_PROXY_ADDRESS_LOCAL: '0x5eb3Bc0a489C5A8288765d2336659EbCA68FCd00',
-  AGENT_REGISTRY_ADDRESS_LOCAL: '0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512',
-  COMPONENT_REGISTRY_ADDRESS_LOCAL:
-    '0x5FbDB2315678afecb367f032d93F642f64180aa3',
-  SERVICE_REGISTRY_ADDRESS_LOCAL: '0x36C02dA8a0983159322a80FFE9F24b1acfF8B570',
-  GENERIC_BOND_CALC_ADDRESS: '0x809d550fca64d94Bd9F66E60752A544199cfAC3D',
+
+const LOCAL_ADDRESSES = {
```

Comment:
Again, does this refactor contribute to performance? The change seems unrelated to the purpose of the PR.